apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -373,6 +374,83 @@ type FlowCollectorIPFIXReceiver struct {
 	Transport string `json:"transport,omitempty"`
 }
 
+type FlowCollectorOpenTelemetryLogs struct {
+	// Set `enable` to `true` to send logs to Open Telemetry receiver.
+	//+kubebuilder:default:=true
+	Enable *bool `json:"enable,omitempty"`
+}
+
+type FlowCollectorOpenTelemetryMetrics struct {
+	// Set `enable` to `true` to send metrics to Open Telemetry receiver.
+	//+kubebuilder:default:=true
+	Enable *bool `json:"enable,omitempty"`
+
+	// Prefix added to each metric name
+	// +optional
+	Prefix string `json:"prefix,omitempty"`
+
+	// How often should metrics be sent to collector
+	// +kubebuilder:default:="20s"
+	PushTimeInterval *metav1.Duration `json:"pushTimeInterval,omitempty"`
+
+	// Time duration of no-flow to wait before deleting data item
+	// +kubebuilder:default:="2m"
+	ExpiryTime *metav1.Duration `json:"expiryTime,omitempty"`
+}
+
+type FlowCollectorOpenTelemetryTraces struct {
+	// Set `enable` to `true` to send traces to Open Telemetry receiver.
+	//+kubebuilder:default:=true
+	Enable *bool `json:"enable,omitempty"`
+
+	// Separate span for each prefix listed
+	//+optional
+	SpanSplitter []string `json:"spanSplitter,omitempty"`
+}
+
+// Add more exporter types below
+type ConnectionType string
+
+type FlowCollectorOpenTelemetry struct {
+	// Address of the Open Telemetry receiver
+	// +kubebuilder:default:=""
+	TargetHost string `json:"targetHost"`
+
+	// Port for the Open Telemetry receiver
+	TargetPort int `json:"targetPort"`
+
+	// Type of Open Telemetry connection. The available options are `http` and `grpc`.
+	// +optional
+	Type string `json:"type"`
```

Comment:
Can we rename `protocol` ?
That's the term used in otel: https://opentelemetry.io/docs/collector/configuration/

Reply:
Yes !